packages/components/src/components/calendar/views/calendar-decades.tsx

```diff
@@ -33,7 +33,7 @@ const Decades = ({ calendar_date, isPeriodDisabled, selected_date, updateSelecte
                             'dc-calendar__cell--disabled': is_disabled,
                             'dc-calendar__cell--other': is_other_century,
                         })}
-                        onClick={is_disabled ? undefined : e => updateSelected(e, 'decade')}
+                        onClick={is_disabled ? undefined : e => updateSelected(e, 'year')}
```

Comment:
@jim-deriv why do we replace 'year' with 'decade' here?😊

Reply:
I'll check this @maryia-deriv

Reply:
@maryia-deriv, So far this is the only place that uses `decade` and in the end , it's replaced with `year` as I mentioned in one of the comments below. I think this is ok, because `decade` is not supported as part of the `moment.unitOfTime.StartOf`, which is the `type` of the parameter for the `toMoment(date).clone().startOf(unit)` that is being used in the calendar

![decade 1](https://user-images.githubusercontent.com/104334373/214740593-fbe6df93-d858-49da-860b-a3247d364dc7.png)

Reply:
@maryia-deriv  This is addressed in the last comment